Repair on migrating to solidui/solidpod after ui migration

```
flutter pub upgrade
Resolving dependencies... (2.3s)
Downloading packages... 
  _fe_analyzer_shared 92.0.0 (94.0.0 available)
  analyzer 9.0.0 (10.0.2 available)
  csv 6.0.0 (7.1.0 available)
  flat_buffers 23.5.26 (25.9.23 available)
  meta 1.17.0 (1.18.1 available)
  petitparser 6.1.0 (7.0.1 available)
  puppeteer 3.19.0 (3.20.0 available)
! solidpod 0.11.0 from git https://github.com/anusii/solidpod.git at 2cac45 (overridden)
! solidui 0.2.0 from git https://github.com/anusii/solidui.git at b64949 (overridden)
  test 1.28.0 (1.29.0 available)
  test_api 0.7.8 (0.7.9 available)
  test_core 0.6.14 (0.6.15 available)
  xml 6.5.0 (6.6.1 available)
No dependencies changed.
11 packages have newer versions incompatible with dependency constraints.
Try `flutter pub outdated` for more information.
flutter run --device-id linux
Launching lib/main.dart on Linux in debug mode...
Building Linux application...                                   
ERROR: lib/features/home/widgets/file_management_content.dart:176:7: Error: No named parameter with the name 'fileTypeResolver'.
ERROR:       fileTypeResolver: healthFileTypeResolver,
ERROR:       ^^^^^^^^^^^^^^^^
ERROR: ../../../../.pub-cache/git/solidui-b649494c2f6282e36c72e157e653e3a879728a50/lib/src/widgets/solid_file.dart:132:9: Context: Found this candidate, but the arguments don't match.
ERROR:   const SolidFile({
ERROR:         ^^^^^^^^^
ERROR: lib/constants/file_type_configs.dart:98:7: Error: No named parameter with the name 'typeId'.
ERROR:       typeId: 'blood_pressure',
ERROR:       ^^^^^^
ERROR: ../../../../.pub-cache/git/solidui-b649494c2f6282e36c72e157e653e3a879728a50/lib/src/models/file_type_config.dart:85:9: Context: Found this candidate, but the arguments don't match.
ERROR:   const FileTypeConfig({
ERROR:         ^^^^^^^^^^^^^^
ERROR: lib/constants/file_type_configs.dart:106:7: Error: No named parameter with the name 'typeId'.
ERROR:       typeId: 'vaccination',
ERROR:       ^^^^^^
ERROR: ../../../../.pub-cache/git/solidui-b649494c2f6282e36c72e157e653e3a879728a50/lib/src/models/file_type_config.dart:85:9: Context: Found this candidate, but the arguments don't match.
ERROR:   const FileTypeConfig({
ERROR:         ^^^^^^^^^^^^^^
ERROR: lib/constants/file_type_configs.dart:114:7: Error: No named parameter with the name 'typeId'.
ERROR:       typeId: 'medication',
ERROR:       ^^^^^^
ERROR: ../../../../.pub-cache/git/solidui-b649494c2f6282e36c72e157e653e3a879728a50/lib/src/models/file_type_config.dart:85:9: Context: Found this candidate, but the arguments don't match.
ERROR:   const FileTypeConfig({
ERROR:         ^^^^^^^^^^^^^^
ERROR: lib/constants/file_type_configs.dart:122:7: Error: No named parameter with the name 'typeId'.
ERROR:       typeId: 'diary',
ERROR:       ^^^^^^
ERROR: ../../../../.pub-cache/git/solidui-b649494c2f6282e36c72e157e653e3a879728a50/lib/src/models/file_type_config.dart:85:9: Context: Found this candidate, but the arguments don't match.
ERROR:   const FileTypeConfig({
ERROR:         ^^^^^^^^^^^^^^
ERROR: lib/constants/file_type_configs.dart:130:7: Error: No named parameter with the name 'typeId'.
ERROR:       typeId: 'profile',
ERROR:       ^^^^^^
ERROR: ../../../../.pub-cache/git/solidui-b649494c2f6282e36c72e157e653e3a879728a50/lib/src/models/file_type_config.dart:85:9: Context: Found this candidate, but the arguments don't match.
ERROR:   const FileTypeConfig({
ERROR:         ^^^^^^^^^^^^^^
ERROR: Target kernel_snapshot_program failed: Exception
Error: Build process failed
make: *** [support/flutter.mk:121: linux] Error 1
```

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -44,14 +44,14 @@ dev_dependencies:
   flutter_lints: ^6.0.0
   import_order_lint: ^0.2.2
   markdown_tooltip: ^0.0.10
-  solidui: ^0.1.0
+  solidui: ^0.2.0
   window_manager: ^0.5.1
 
 dependency_overrides:
   solidui:
     git:
       url: https://github.com/anusii/solidui.git
-      ref: tony/59_ui_migration
+      ref: dev
 
 flutter:
   assets:

--- a/pubspec.yaml
+++ b/pubspec.yaml
@@ -51,7 +51,7 @@ dependency_overrides:
   solidui:
     git:
       url: https://github.com/anusii/solidui.git
-      ref: dev
+      ref: tony/153_refactor_file_browsing_code
 
 flutter:
   assets:

